### PR TITLE
[ADD] l10n_de_country_states addon

### PR DESCRIPTION
**Pull request by antespi**

### CODE REVIEW

**Comment by antespi** [comment]
Any news?

File: l10n_de_country_states/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    ],
+    'update_xml': [
+        'data/res_country_states.xml'
+    ],
+    'test': [
+    ],
+    'demo_xml': [
+    ],
+    'installable': True,
+    'active': False,
```
Comment (dreispt) [inline l10n_de_country_states/__openerp__.py]: `active` attribute is no longer used

File: l10n_de_country_states/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    ],
+    'init_xml': [
+    ],
+    'update_xml': [
+        'data/res_country_states.xml'
+    ],
+    'test': [
+    ],
+    'demo_xml': [
+    ],
```
Comment (dreispt) [inline l10n_de_country_states/__openerp__.py]: remove init_xml, test and demo_xml keys

File: l10n_de_country_states/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    'version': '1.0',
+    'author': 'initOS GmbH & Co. KG, '
+              'Antiun Ingeniería S.L.',
+    'website': '',
+    'depends': [
+        'base',
+    ],
+    'init_xml': [
+    ],
+    'update_xml': [
```
Comment (dreispt) [inline l10n_de_country_states/__openerp__.py]: 'data' instead of 'update_xml'

**Comment by dreispt** [comment]
Minor fix suggestions, but :+1:

File: l10n_de_country_states/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    'name': 'Country States',
```
Comment (dreispt) [inline l10n_de_country_states/__openerp__.py]: Maybe improve to something like "Germany Country States"

File: l10n_de_country_states/i18n/de.po [hunk 5]
```diff
@@ -0,0 +1,102 @@
+# Translation of OpenERP Server.
+# This file contains the translation of the following modules:
+#	* l10n_de_country_states
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: OpenERP Server 7.0\n"
+"Report-Msgid-Bugs-To: \n"
```
Comment (ntodorova) [inline l10n_de_country_states/i18n/de.po:8]: The Project-Id-Version is old

File: l10n_de_country_states/i18n/l10n_de_country_states.pot [hunk 6]
```diff
@@ -0,0 +1,102 @@
+# Translation of OpenERP Server.
+# This file contains the translation of the following modules:
+#	* l10n_de_country_states
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: OpenERP Server 7.0\n"
+"Report-Msgid-Bugs-To: \n"
```
Comment (ntodorova) [inline l10n_de_country_states/i18n/l10n_de_country_states.pot:8]: The Project-Id-Version is old

**Comment by ntodorova** [comment]
:+1:

**Pull request merged**